dashboard/app/views/admin_users/permissions_form.html.haml

```diff
@@ -55,7 +55,8 @@
   %button.btn{type: 'submit'}
     %i.fa.fa-search
 -if @users_with_permission
-  = page_header 'users', @users_with_permission, permitted_params: [:permission]
+  - download_csv = link_to('Page as CSV', permissions_csv_path(params.permit([:permission, :page_size, :page, :search_term])), class: 'btn btn-default')
```

Comment:
This button text might be more intuitive as "Download CSV"  or "Download Page as CSV".

Reply:
Yeah, I appreciate this comment... I had "Download Page as CSV" at one point and it felt comically long and would get in the way of adding any buttons to that header in the future. And just "Download CSV" might be surprising that the CSV is just the subset on the screen. It's not a user-facing button, so after thinking it through, I think I'll keep it as "Page as CSV" and not overthink it too much.

Reply:
Plus I'm a tad lazy to think too much about a feature for an internal tool. :)